src/Ryujinx.Graphics.Gpu/Image/AutoDeleteCache.cs

```diff
@@ -56,6 +60,19 @@ class AutoDeleteCache : IEnumerable<Texture>
 
         private readonly Dictionary<TextureDescriptor, ShortTextureCacheEntry> _shortCacheLookup;
 
+        /// <summary>
+        /// Gets MaxTextureCapacity Dynamically
```

Comment:
```suggestion
        /// Initializes the cache, setting the maximum texture capacity for the specified GPU context.
```
This needs to be updated since the function changed.
BTW, XML docs for the new `context` parameter is missing.